packaging/convert2rhel.spec

```
@@ -124,6 +124,27 @@ install -m 0600 config/convert2rhel.ini %{buildroot}%{_sysconfdir}/convert2rhel.
 %attr(0644,root,root) %{_mandir}/man8/%{name}.8*
 
 %changelog
+* Mon Feb 20 2023 Eric Gustavsson <egustavs@redhat.com> 1.2
+- Remove python 2.6 and RHEL 6 conversion capability
+- Check for kernel boot files after conversion
+- Allowing the overriding of kmod inhibitors
+- Remove shim-x64 special workaround
+- Always print the RHSM rollback task "title"
+- Check if org and key is passed
+- Improve logging and handling of RHSM facts
+- Change the label for FILE output to DEBUG
+- Add progress indicators for YUM and DNF modules
+- Refactor yum makecache to be separate from main checks
```

```suggestion
- Move clean_yum_metadata() to the pkgmanager module
```
The original PR https://github.com/oamg/convert2rhel/pull/724 title was not updated after the purpose of the PR changed.

---

This can be omitted from the changelog as it is a minor code cleaning item.